crates/rpc/src/method/simulate_transactions.rs

```diff
@@ -1087,15 +1142,17 @@ pub(crate) mod tests {
                 last_block_header: &BlockHeader,
                 universal_deployer_address: ContractAddress,
             ) -> pathfinder_executor::types::TransactionSimulation {
+                let overall_fee_correction = 1u64;
```

Comment:
wonder where this comes from... :-)

Reply:
I _think_ that's because our assumption that you can subtract the L1 gas cost of validation from the overall fee is not valid. A single Cairo step is 0.0025 L1 gas, so due to rounding you cannot compute the overall cost by simply adding the rounded values.